.github/workflows/main.yml

```diff
@@ -1,5 +1,5 @@
 name: CI
-on: [push]
+on: [pull_request, push]
```

Comment:
When Gurbir initially switched the CI to GitHub Actions, I ended up doing some research on it, concerned about it being (ab)used as an attack vector.  I believe that my conclusion was that just `push` was correct...but, I no longer recall why.  🙁  (It's possible that I wasn't taking into account forked repos at that time.)  I hope this does what we want it to (and nothing else...).

Reply:
I'd done research before that when I set up GitHub Actions on the pyesbulk repo, and at the time decided that I needed both, though I no longer recall the details of why; understanding precisely the conditions and context of the webhook events proves extraordinarily complex because each segment of the documentation is fairly shallow and doesn't provide the vertical context I want.

I would have expected that `push` would track pushing a commit to a PR branch; however, it clearly does not, and  `pbench-dashboard` seems to be currently running the Action only when a PR is merged into `main`, which isn't what we want.

The [documentation](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request) suggests, in a fairly convoluted and inconclusive way, that `pull_request_target` may be less of a potential exposure, but basically goes on to point out exactly the same potential issues as `pull_request` and I don't see any more appropriate target.

Reply:
> I would have expected that push would track pushing a commit to a PR branch

Without brushing up on the documentation, I would expect the same but only if it were pushed to the same repo and not to a fork of it....